### DISCUSSION TITLE
GitHub etiquette question:  is a request for help an Issue?

### DISCUSSION THREAD

**Discussion by funnypolynomial** [community/community#22908] [New to GitHub]
<p>Someone has created an Issue on an Arduino repo of mine, asking “What does this do?/I want to do a thing./Will this do it?/How do I write it?” etc.  Is it appropriate to have an on-going dialog on GitHub, under the Issue, or do I suggest taking it offline, exchange emails directly?  I do not expect any code changes to come of it.</p>
<p>thanks,</p>
<p>Mark</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/funnypolynomial">@funnypolynomial</a>,</p>
<p>How issues are used is generally up to the maintainer or the community that work on a particular repository. Some people like to keep all communication about a project in GitHub and use issues to discuss all aspects of the project, including long term strategy. Other people want to keep issues solely for bug tracking.</p>
<p>No matter how you use issues, it can be helpful to use labels to categorize different types of issues such as discussions, feature requests, and bug reports so that you can better track and respond to these various types of issues.</p>
<p>For more information, you may be interested in the GitHub Guide, <a href="https://guides.github.com/features/issues/" rel="nofollow noopener">Mastering Issues</a>.</p>
<p>Thanks!</p>
